harpy/impute.py

```
@@ -9,17 +9,17 @@
 from ._cli_types_generic import convert_to_int, HPCProfile, InputFile, SnakemakeParams
 from ._cli_types_params import StitchParams
 from ._conda import create_conda_recipes
-from ._launch import launch_snakemake, SNAKEMAKE_CMD
-from ._misc import fetch_rule, fetch_report, fetch_script, snakemake_log
-from ._parsers import parse_alignment_inputs, biallelic_contigs
-from ._printing import workflow_info
+from ._launch import launch_snakemake
+from ._misc import fetch_rule, fetch_report, fetch_script, snakemake_log, write_snakemake_config, write_workflow_config
```

Ruff (0.8.2)

`._misc.fetch_script` imported but unused

Remove unused import: `._misc.fetch_script`

(F401)